tests/unit/test_topic_visualizer.py

```diff
@@ -0,0 +1,80 @@
+
+import gensim
+import pytest
+import spacy
+import pandas
+
+from sem_covid.services.sc_wrangling.data_cleaning import clean_text_from_specific_characters
+from sem_covid.entrypoints.notebooks.topic_modeling.topic_modeling_wrangling.topic_visualizer import TopicInformation
+from sem_covid.entrypoints.notebooks.topic_modeling.topic_modeling_wrangling.lda_modeling import WordsModeling
+
+nlp = spacy.load("en_core_web_sm")
+
+
+class TestTopicVisualizer:
+
+    @pytest.fixture(autouse=True)
+    def _insert_transformed_pwdb_dataframe(self, transformed_pwdb_dataframe):
+        self._transformed_pwdb_dataframe = transformed_pwdb_dataframe
+
+    def _instantiate_topic_information(self):
+        pwdb_descriptive_data = self._transformed_pwdb_dataframe['title'].map(str) + ' ' + \
+                                self._transformed_pwdb_dataframe['background_info_description'].map(str) + ' ' + \
+                                self._transformed_pwdb_dataframe['content_of_measure_description'].map(str) + ' ' + \
+                                self._transformed_pwdb_dataframe['use_of_measure_description'] + ' ' + \
+                                self._transformed_pwdb_dataframe['involvement_of_social_partners_description']
+
+        unused_characters = ["\\r", ">", "\n", "\\", "<", "''", "%", "...", "\'", '"', "(", "\n", "[", "]"]
+        clean_text = clean_text_from_specific_characters(pwdb_descriptive_data, unused_characters)
+        doc = nlp(clean_text)
+
+        noun_phrases = [word.text for word in doc.noun_chunks]
+        separated_noun_phrases = [[nouns] for nouns in noun_phrases]
+
+        word_modeling = WordsModeling(separated_noun_phrases)
```

Comment:
really words?

Reply:
because this class is used not only for noun phrases